alignak_backend/livestate.py

```diff
@@ -95,15 +97,19 @@ def on_inserted_service(items):
                     'last_state': 'OK', 'last_state_type': 'HARD', 'output': '',
                     'long_output': '', 'perf_data': '', 'type': 'service',
                     'business_impact': item['business_impact'], 'display_name_service': name,
-                    'display_name_host': name_h, '_realm': item['_realm']}
+                    'display_name_host': name_h, '_realm': item['_realm'],
+                    'name': host_info['name'] + '/' + item['name']}
             if item['initial_state'] == 'w':
```

Comment:
We have _display_name_host_, why not change _name_ in _display_service_name_ and put only the service name?